tests/foreman/data/zoo_custom_gpgkey.txt

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
We are keeping this key in our robottelo framework under  `tests/foreman/data/`. Packages under custom fake repos, that we are using in our automation, are signed by this key.

We can generate our own key by `gpg --gen-key` but packages must be signed with that then only we can use that.